com.maplerain.telsis_translator_flutter.metainfo.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You should maintain this file in your repo and distribute it in the tarball.

Reply:
Yes, it is in the tarball. I saw this file on other repos and blindly followed. My apologies. 🙇‍♀️